scripts/build_funcs.sh

```diff
@@ -23,32 +25,32 @@ GIT_VERSION=$(git describe --always --dirty --tags)
 IMAGE_VERSION=$(git describe --always --dirty --tags)
 IMAGE_TAG=$(echo "$IMAGE_VERSION" | cut -d. -f1,2)
 IMAGE_BRANCH=$(git rev-parse --abbrev-ref HEAD | sed 's/\///g')
-GIT_REF_SHORT=$(git rev-parse --short=8 --verify HEAD)
 GIT_REF_LONG=$(git rev-parse --verify HEAD)
 
 BUILDMNT=/go/src/$GOTARGET
 BUILD_IMAGE=golang:1.16
 AMD_IMAGE=gcr.io/distroless/static:nonroot
 ARM_IMAGE=gcr.io/distroless/static:nonroot-arm64
-WIN_IMAGE=mcr.microsoft.com/windows/servercore:1809
+WIN_AMD64_BASEIMAGE=mcr.microsoft.com/windows/nanoserver
```

Comment:
Not sure if this matters. @perithompson do you have an opinion? In your blog post I saw both and I just went with nanoserver for size alone.